app/components/primer/beta/button.html.erb

```diff
@@ -8,7 +8,12 @@
     <span class="Button-label"><%= trimmed_content %></span>
     <% if trailing_visual %>
       <span class="Button-visual Button-trailingVisual">
-        <%= trailing_visual %>
+        <% if @trailing_visual_counter %>
+          <span class="sr-only">(<%= trailing_visual %>)</span>
+          <span aria-hidden="true"><%= trailing_visual %></span>
```

Comment:
Why is this causing a shift 🤔 

<img width="96" alt="Screenshot of visual counter a few pixels down because of this change" src="https://github.com/primer/view_components/assets/16447748/f5807340-1b56-421e-9ec1-a19e655cc4b6">

Reply:
What I'm [seeing in my browser](http://primer-view-components-preview-2807.eastus.azurecontainer.io/view-components/lookbook/inspect/primer/beta/button/trailing_counter?_display=%257B%2522theme%2522%253A%2522light%2522%257D) does not match the changes in the snapshots. Does anyone know why I might not be seeing what's reflected in the snapshots?

Reply:
Ok actually nvm I see it now.

Having the counter nested inside another `<span>` seems to be the issue. Looks like I might want to directly set the `aria-hidden` on the counter.